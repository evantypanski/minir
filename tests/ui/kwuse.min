func main() -> none {
  let func = 0;
  let let = 0;
  let ret = 0;
  let br = 0;
  let brc = 0;

  let none = 0;
  let int = 0;
  let float = 0;
  let boolean = 0;

  let debug = 0;
  let alloc = 0;

  let true = 0;
  let false = 0;
  let undefined = 0;

  ret;
}
