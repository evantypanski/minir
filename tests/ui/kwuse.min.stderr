error: at tests/ui/kwuse.min:2: 'func' is a keyword and cannot be used as an identifier
  let func = 0
error: at tests/ui/kwuse.min:3: 'let' is a keyword and cannot be used as an identifier
  let let = 0
error: at tests/ui/kwuse.min:4: 'ret' is a keyword and cannot be used as an identifier
  let ret = 0
error: at tests/ui/kwuse.min:5: 'br' is a keyword and cannot be used as an identifier
  let br = 0
error: at tests/ui/kwuse.min:6: 'brc' is a keyword and cannot be used as an identifier
  let brc = 0
error: at tests/ui/kwuse.min:8: 'none' is a keyword and cannot be used as an identifier
  let none = 0
error: at tests/ui/kwuse.min:9: 'int' is a keyword and cannot be used as an identifier
  let int = 0
error: at tests/ui/kwuse.min:10: 'float' is a keyword and cannot be used as an identifier
  let float = 0
error: at tests/ui/kwuse.min:11: 'boolean' is a keyword and cannot be used as an identifier
  let boolean = 0
error: at tests/ui/kwuse.min:13: 'debug' is a keyword and cannot be used as an identifier
  let debug = 0
error: at tests/ui/kwuse.min:14: 'alloc' is a keyword and cannot be used as an identifier
  let alloc = 0
error: at tests/ui/kwuse.min:16: 'true' is a keyword and cannot be used as an identifier
  let true = 0
error: at tests/ui/kwuse.min:17: 'false' is a keyword and cannot be used as an identifier
  let false = 0
error: at tests/ui/kwuse.min:18: 'undefined' is a keyword and cannot be used as an identifier
  let undefined = 0


Found 14 errors during parsing
error: Fatal error occurred; aborting
